admin/class-schema-person-upgrade-notification.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Notifies the user to update the Person on the publish entity in the Configuration Wizard.
+ */
+class WPSEO_Schema_Person_Upgrade_Notification implements WPSEO_WordPress_Integration {
+
+	/**
+	 * Registers all hooks to WordPress
+	 *
+	 * @return void
+	 */
+	public function register_hooks() {
+		add_action( 'admin_init', array( $this, 'handle_notification' ) );
+	}
+
+	/**
+	 * Handles if the notification should be added or removed.
+	 */
+	public function handle_notification() {
+		if ( WPSEO_Options::get( 'company_or_person' ) === 'person' && empty( WPSEO_Options::get( 'company_or_person_user_id', false ) ) ) {
```

Comment:
The build fais. Looks like phpcs isn't happy with this:
```
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 26 | ERROR | Only variables can be passed to empty() prior to PHP
    |       | 5.5.
```

Also, `WPSEO_Options::get( 'company_or_person_user_id', false )` returns false or an integer (the user id), not sure the check for `empty()` is correct.